python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -100,10 +130,53 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
     )
 
 
+def transform_sql(
+    sql: str,
+    conn_id: str = "",
+    parameters: dict | None = None,
+    database: str | None = None,
+    schema: str | None = None,
+    **kwargs: Any,
+) -> XComArg:
+    """
+    This function returns a ``Table`` object that can be passed to future tasks from specified SQL code.
+    Tables can be inserted via the parameters kwarg.
+
+    :param sql: The sql code to execute. You can either pass raw sql or the path to a sql file. Can be an absolute path,
+        or you can use a relative path if the `template_searchpath` variable is set in your DAG.
+    :param conn_id: Connection ID for the database you want to connect to. If you do not pass in a value for this object
+        we can infer the connection ID from the first table passed into the python_callable function.
+        (required if there are no table arguments)
+    :param parameters: parameters to pass into the SQL query
+    :param database: Database within the SQL instance you want to access. If left blank we will default to the
+        table.metatadata.database in the first Table passed to the function (required if there are no table arguments)
+    :param schema: Schema within the SQL instance you want to access. If left blank we will default to the
+        table.metatadata.schema in the first Table passed to the function (required if there are no table arguments)
+    :param kwargs: Any keyword arguments supported by the BaseOperator is supported (e.g ``queue``, ``owner``)
+    :return: Transform functions return a ``Table`` object that can be passed to future tasks.
+        This table will be either an auto-generated temporary table,
+        or will overwrite a table given in the `output_table` parameter.
+    """
+
+    def null_function():
+        return sql, parameters
+
+    return TransformOperator(
+        conn_id=conn_id,
+        parameters=parameters,
+        handler=None,
+        database=database,
+        schema=schema,
+        sql=sql,
+        python_callable=null_function,
```

Comment:
```suggestion
        python_callable=lambda: sql, parameters,
```

Reply:
Seems to work 🤔 I thought I had tried that. I remember that there was a type issue with lambda at least - that is why I changed that.